doc/index.md

```diff
@@ -6,6 +6,21 @@ title: Entity Framework 6
 Npgsql has an Entity Framework 6 provider. You can use it by installing the
 [EntityFramework6.Npgsql](https://www.nuget.org/packages/EntityFramework6.Npgsql/) nuget.
 
+## Basic Usage ##
```

Comment:
This section seems to be more about configuration than usage. Propose changing usage to configuration.

Reply:
resolved.